huggle/hugglequeuefilter.hpp

```diff
@@ -113,6 +119,9 @@ namespace Huggle
             HuggleQueueFilterMatch Self;
             HuggleQueueFilterMatch UserSpace;
             HuggleQueueFilterMatch TalkPage;
+            QString Type;
```

Comment:
Looked more into this I don't really understand what is point of all these 3 variables? Filter is only 1 for whole queue, so list of categories you want to exclude, or require should be defined in each filter separately. What are these 3 variables for? And how do you store a list of categories that user wants to filter now?